Sources/EduIDExpansion/Flows/Home/HomeNavigationDelegate.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+protocol HomeNavigationDelegate: AnyObject {
```

Comment:
See https://github.com/Tiqr/tiqr-app-core-ios/pull/38#discussion_r1100025159

Reply:
I did this